src/CodeStyle/Core/Analyzers/Options/OptionSetExtensions.cs

```diff
@@ -19,16 +19,22 @@ public static T GetOption<T>(this OptionSet optionSet, Option<T> option)
             return value;
         }
 
-        public static T GetOption<T>(this OptionSet optionSet, PerLanguageOption<T> option, string language)
+        public static T GetOption<T>(this OptionSet optionSet, PerLanguageOption<T> option)
         {
             if (!TryGetEditorConfigOption(optionSet, option, out T value))
-            {
                 value = option.DefaultValue;
-            }
 
             return value;
         }
 
+        public static T GetOption<T>(this OptionSet optionSet, PerLanguageOption<T> option, string language)
```

Comment:
Why do you need this method? Are there too many callsites to be adjusted to remove the language argument?

Reply:
It's needed for cases where we still conditionally use `AnalyzerConfigOptions` for Code Style and `OptionSet` for Workspaces.

Reply:
I am assuming we want to eventually remove all such cases, at which point this can go away?

Reply:
Yes that's what I'm hoping.